Proposed simpler PR template (#19)
pi-hole/.github: .github/PULL_REQUEST_TEMPLATE.md

## Thank you for your contribution to the Pi-hole Community! 

Please read the comments below to help us consider your Pull Request.

We are all volunteers and completing the process outlined will help us review your commits quicker.

> **Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the `development` branch.**

Reviewer: This template is used both for the `core` as well as the `web` repo. However, `development` only exists on `core`, on `web` it's named `devel`. Do we need to mention this?

Author: We could remove the codeblock from `development` and say 

**PR's against the repository developmental branch.**

Reviewer: Sounds good to me.

> **Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the repositories developmental branch.**

Reviewer: ```suggestion
**Please make sure you
 1. base your code and PRs against the repositories developmental branch.
 2. [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits as we enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions**
```

> We enforce the [DCO](https://docs.pi-hole.net/guides/github/contributing/) for all contributions.

Reviewer: ```suggestion
We enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions. Please [sign all your commits](https://docs.pi-hole.net/guides/github/how-to-signoff/).
```

Author: I think that will be confusing to have the DCO and the commit signoffs so close to each other.

> We enforce the [DCO](https://docs.pi-hole.net/guides/github/contributing/) for all contributions.

Reviewer: ```suggestion

```

> We enforce the [DCO](https://docs.pi-hole.net/guides/github/contributing/) for all contributions.

Reviewer: ```suggestion

```

---
- **What does this PR aim to accomplish?:**

*Replace this with detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues*

- **How does this PR accomplish the above?:**

*Replace this with a detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix*

- **What documentation changes (if any) are needed to support this PR?:**

*Replace this with a detailed list of any necessary changes*

---
**By submitting this pull request, I confirm the following:** 

> 1. I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 

Reviewer: ```suggestion
1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
```
